Scripts/Modules/Highlight/HighlightModule.cs

```diff
@@ -17,11 +19,10 @@ void LateUpdate()
 			if (go == null)
 				continue;
 
-			var highlightTransform = go.transform;
-			var highlightScaleIncrease = Vector3.one * 0.0125f;
-			Matrix4x4 highlightScaleIncreaseMatrix = Matrix4x4.TRS(highlightTransform.position, highlightTransform.rotation, highlightTransform.lossyScale + highlightScaleIncrease);
 			foreach (var m in go.GetComponentsInChildren<MeshFilter>())
 			{
+				var highlightTransform = m.transform;
```

Comment:
Yes. It was creating the transformation matrix once for the parent object, thus it was drawing all objects in the same position/rotation/scale. This moves the TRS call into the foreach loop, and turns the scale increase into a static/readonly field which makes it a compile-time constant. Not sure if this makes a huge difference but stylistically I like to have those tweakable hardcoded values defined in the class scope.